.github/workflows/check-release.yml

```diff
@@ -36,17 +40,25 @@ jobs:
       package_name: ${{ steps.version_check.outputs.package_name }}
       package_version: ${{ steps.version_check.outputs.package_version }}
 
-  prepare-release:
-    needs: [check-version]
+  test-kedro-viz:
```

Comment:
qq - what is this for?

Reply:
@rashidakanchwala  Ref: https://github.com/kedro-org/kedro-viz/issues/1785#issuecomment-1979635461
Previously We run all the above jobs in parallel which means even if the tests fail for some reason, we still tag our release and publish it to npm, pypi. this change will runs all the required tests (lint, unit, e2e) -> tags the release -> releases it to pypi and npm in a sequence.